curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-data-structures/add-key-value-pairs-to-javascript-objects.english.md

```diff
@@ -7,37 +7,54 @@ forumTopicId: 301153
 
 ## Description
 <section id='description'>
-At their most basic, objects are just collections of <dfn>key-value pairs</dfn>, or in other words, pieces of data mapped to unique identifiers that we call <dfn>properties</dfn> or <dfn>keys</dfn>. Let's take a look at a very simple example:
+At their most basic, objects are just collections of <dfn>key-value</dfn> pairs. In other words, its pieces of data (<dfn>values</dfn>) mapped to unique identifiers called <dfn>properties</dfn> (<dfn>keys</dfn>). Take a look at an example:
 
 ```js
-let FCC_User = {
-  username: 'awesome_coder',
-  followers: 572,
-  points: 1741,
-  completedProjects: 15
+const tekkenCharacter = {
+  player: 'Hwoarang',
+  fightingStyle: 'Tae Kwon Doe',
+  human: true
 };
 ```
 
-The above code defines an object called <code>FCC_User</code> that has four <dfn>properties</dfn>, each of which map to a specific value. If we wanted to know the number of <code>followers</code> <code>FCC_User</code> has, we can access that property by writing:
+The above code defines a Tekken video game character object called <code>tekkenCharacter</code>. It has three <dfn>properties</dfn>, each of which map to a specific <dfn>value</dfn>. If you want to add an additional property, such as "origin", it can be done by assigning <code>origin</code> to the object:
 
 ```js
-let userData = FCC_User.followers;
-// userData equals 572
+tekkenCharacter.origin = 'South Korea';
 ```
 
-This is called <dfn>dot notation</dfn>. Alternatively, we can also access the property with brackets, like so:
+This uses <dfn>dot notation</dfn>. If you were to observe the <code>tekkenCharacter</code> object, it will now include the <code>origin</code> property. Hwoarang also had distinct orange hair. You can add this property with <dfn>bracket notation</dfn> by doing:
 
 ```js
-let userData = FCC_User['followers'];
-// userData equals 572
+tekkenCharacter['hair color'] = 'dyed orange';
+```
+
+<dfn>Bracket notation</dfn> is required if your property has a space in it or if you want to use a variable to name the property. In the above case, the property is enclosed in quotes to denote it as a string and will be added exactly as shown. Without quotes, it will be evaluated as a variable and the name of the property will be whatever value the variable is. Here's an example with a variable:
```

Comment:
```suggestion
Bracket notation is required if your property has a space in it or if you want to use a variable to name the property. In the above case, the property is enclosed in quotes to denote it as a string and will be added exactly as shown. Without quotes, it will be evaluated as a variable and the name of the property will be whatever value the variable is. Here's an example with a variable:
```